src/main/java/io/confluent/connect/jdbc/JdbcSourceConnector.java

```diff
@@ -104,8 +105,8 @@ public void start(Map<String, String> properties) throws ConnectException {
       // query.
       whitelistSet = Collections.emptySet();
     }
-    tableMonitorThread = new TableMonitorThread(db, context, tablePollMs, whitelistSet,
-                                                blacklistSet, tableTypesSet);
+    tableMonitorThread = new TableMonitorThread(db, schemaPattern, context, tablePollMs,
```

Comment:
@pqueixalos sorry didn't notice this minor thing previously, but in terms of arg ordering `schemaPattern` makes sense after the context